### DISCUSSION TITLE
GitHub Repository

### DISCUSSION THREAD

**Discussion by 2199105T** [IBAHCM/RPiR#348] [Archive]
Hi @richardreeve @soniamitchell 

Sorry to bother you, but I am trying to keep up the habit of using Github as you suggested for my other classes.

I created a new private repository on GitHub but when I go to File > New Project > Version Control > Git Repository, I get this  pop up and I can't find a straight answer online of why this is happening so I was wondering if you could point me in the right direction?

Here is the pop up message:
<img width="441" alt="Screenshot 2021-03-03 at 17 13 15" src="https://user-images.githubusercontent.com/77625039/109845311-e6fea300-7c44-11eb-9ab2-f73a7c5d1d22.png">

Thanks,

Sarah

**Comment by soniamitchell** [comment]
Hello! Have you updated your OS lately? Could you try running `xcode-select --install` in the terminal? Then try again.

**Reply by 2199105T** [reply]
Hi @soniamitchell !

I ran that code in the terminal and installed it etc, it is still coming up with the same message when setting up version control :/

**Reply by soniamitchell** [reply]
If you go to RStudio > Preferences, in the Git/SVN tab, does it look like this?

<img width="651" alt="Screenshot 2021-03-03 at 18 29 57" src="https://user-images.githubusercontent.com/8394297/109854106-8e340800-7c4e-11eb-9dc1-e756e3856798.png">

**Reply by 2199105T** [reply]
Yes it does!

![image](https://user-images.githubusercontent.com/77625039/109854438-f682e980-7c4e-11eb-89b6-ef069db901a5.jpeg)

**Reply by soniamitchell** [reply]
What happens when you uncheck `Enable version control interface for RStudio projects`, then restart RStudio, then check it again?

**Reply by 2199105T** [reply]
It doesn't change anything and still comes up with the pop up when trying version control 🤔

**Reply by soniamitchell** [reply]
I think you might have to reinstall git... 😕

**Reply by 2199105T** [reply]
Okay I'll try that and hopefully it will work! Thank you Sonia 😊

**Reply by soniamitchell** [reply]
Good luck! (let me know how it goes)
